src/main/frontend/util.cljc

```diff
@@ -219,9 +219,9 @@
                                   %))
                           (string/join)
                           (str "#")))]
-       (when (string/starts-with? bg-color "rgb(")
+       (when (string/starts-with? bg-color "rgb")
          (let [rgb (-> bg-color
-                       (string/replace #"^rgb\(" "")
+                       (string/replace #"^rgb[^\d]+" "")
```

Comment:
Is the left bracket ignored?

Reply:
Yes. The left bracket is included.
![CleanShot 2024-04-29 at 16 57 58@2x](https://github.com/logseq/logseq/assets/1779837/52e4d4c7-ef11-490a-a36a-4a3fd87b75de)